src/borg/helpers/fs.py

```diff
@@ -39,7 +39,7 @@ def ensure_dir(path, mode=stat.S_IRWXU, pretty_deadly=True):
             raise Error(e.args[1])
         else:
             raise
-    return True
+    return
```

Comment:
you don't need to have an empty return at the end of a function. if there is nothing left, it'll return anyway.